models/CommentsManager.php

```diff
@@ -79,16 +89,28 @@ public function addComment($userId, $comment, $postId){
 		}else{
 			return  false;
 		}
-
 	}
 
-	public function deleteComment($id){
+	/**
+	 * Supprime un commentaire.
+	 *
+	 * @param int $id
+	 */
+	public function deleteComment(int $id):void {
 
 		$this->getBdd()->exec('DELETE FROM comment WHERE id = ' . (int) $id);
 
 	}
 
-	public function updateComment(string $content,int $id){
+	/**
+	 * Enregistre la modification d'un commentaire.
+	 *
+	 * @param string $content
+	 * @param int $id
+	 *
+	 * @return bool
+	 */
+	public function updateComment(string $content,int $id): bool {
```

Comment:
Codacy found an issue: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)